### ISSUE TITLE
Formatting table doesn't format *correctly*, depending on the characters in cells

### ISSUE THREAD

**Issue by Samonitari** [takumisoft68/vscode-markdown-table#20]
Characters, like `ű` and `ő` is not handled correctly in tables.
These takes up one vertical position, but table is formatted like these were two.

Here is a "formatted" table for demonstration
```
|                                   |     |
| --------------------------------- | --- |
| Gyakorlatvezető                  | XYZ |
| Gyakorlatvezető Gyakorlatvezető | XYZ |
| Tanuló                            | XYZ |
```

Thanks for the extension by the way!
It is such a sad and boring cliché (and I presume it is due to some lib, not your code) that the code doesn't handle well non-ASCII chars. It has a certain humour that - again I presume - it works well on Japanese?

**Comment by takumisoft68** [comment]
@Samonitari 
Thank you for using this ext., and reporting the issue.

That's my oversight with distinguish zenkaku.
I will work on it on next weekend.

**Comment by takumisoft68** [comment]
@Samonitari 
Thank you for using this extension.

I have fixed up width handling at 0.5.2.
